### ISSUE TITLE
Excuse me for interrupting, could you please provide the pytorch training weight file for the accuracy achieved by MSCA-MSDC in the paper？

### ISSUE THREAD

**Issue by 1485995573** [licuiling-chd/EMNet#1]
Excuse me for interrupting, could you please provide the pytorch training weight file for the accuracy achieved by MSCA-MSDC in the paper？
